src/30_Advanced/SeatMap.html

```diff
@@ -0,0 +1,224 @@
+<!---{
+  "preview": "default",
+  "default": "preview"
+}--->
+<!-- ## Introduction -->
+<!--
+This is a sample showing how to implement a seatmap that can be used for theaters, cinemas, airplanes and more.
```

Comment:
Can you explain a bit more what this sample demonstrates? There are so many things going on:

* pan & zoom
* availability
* dynamic seat selection
* ...

Reply:
done